contracts/cw-ics721-bridge/src/contract.rs

```diff
@@ -345,10 +360,8 @@ fn query_nft_contract_for_class_id(deps: Deps, class_id: String) -> StdResult<Op
     CLASS_ID_TO_NFT_CONTRACT.may_load(deps.storage, ClassId::new(class_id))
 }
 
-fn query_metadata(deps: Deps, class_id: String) -> StdResult<Option<String>> {
-    Ok(CLASS_ID_TO_CLASS_URI
-        .may_load(deps.storage, ClassId::new(class_id))?
-        .flatten())
+fn query_metadata(deps: Deps, class_id: String) -> StdResult<Option<Class>> {
+    CLASS_ID_TO_CLASS.may_load(deps.storage, ClassId::new(class_id))
 }
```

Comment:
the return type annotation for `QueryMsg::Metadata {}` needs to be updated to return `Option<Class>` as well